### ISSUE TITLE
Can't add myself to a company

### ISSUE THREAD

**Issue by maxbeizer** [nashdev/jobs#163]
To create a job listing, the app is requiring I create a company. The company I work for already exists in the system. I should be able to add myself to an existing company and then add a job

**Comment by egdelwonk** [comment]
Yeah this is definitely something that I’d like to add in. I guess the question I have would be around moderation of joining a company. Would the original owner and then subsequent members approve joining ? Should it be a free for all? What do you think ?

**Comment by jarroda** [comment]
Yeah, I'm having the same issue, except this is a company that I created 2 years ago, but somehow I don't have access to it now.  It should have been the same user I am using now, but I'm apparently not the owner.  How can I regain ownership of this existing company so I can post a new job opening?

**Comment by egdelwonk** [comment]
Hey @jarroda  -- If the user you created previously (2 years ago) didn't use the exact same e-mail as what you use to login to Nashdev slack, then they wont match. Basically I'd just need to associate your login to those listings. When you login with your slack account, and if the e-mails don't match, a new user is generated at login time, which is what causes the mismatch.

**Comment by jarroda** [comment]
@egdelwonk Yep I believe that is the issue.  I guess the Jobs app from 2 years ago had its own authentication, and I had created a user with a different email than I use to login into NashDev slack.  Now that the Jobs app uses Slack login, I don't have any way to access this company.  Can I have you associate my company with my slack email?  If you can send me a DM on slack, I can give you the information. My NashDev slack user is `jarroda`.
